external/bluetooth/src/bluetooth-adapter.c

```diff
@@ -0,0 +1,2311 @@
+/*
+ * Copyright (c) 2011 Samsung Electronics Co., Ltd All Rights Reserved
```

Comment:
Please make 2011 to 2019 and please check this for all file headers.

Reply:
Same as above.